### PR TITLE
MCOL-4193: Delete or remove from build, unused OAM and applications ProcMon, ProcMgr

### PR DESCRIPTION
**Pull request by benthompson15**

### CODE REVIEW

File: oamapps/columnstoreSupport/mcsSupportUtil.cpp [hunk 1]
```diff
@@ -304,7 +303,7 @@ void getStorageConfig(FILE * pOutputFile)
         }
 
         //get any unassigned DBRoots
-        DBRootConfigList undbrootlist;
+        /*DBRootConfigList undbrootlist;
```
Comment (drrtuy) [inline oamapps/columnstoreSupport/mcsSupportUtil.cpp:306]: Shouldn't we just remove the lines @benthompson15 ?

File: oamapps/columnstoreSupport/mcsSupportUtil.cpp [hunk 2]
```diff
@@ -471,7 +470,7 @@ void getStorageConfig(FILE * pOutputFile)
                     try
```
Comment (drrtuy) [inline oamapps/columnstoreSupport/mcsSupportUtil.cpp:470]: This block looks useless w/o glusterctl call.

File: oamapps/columnstoreSupport/mcsSupportUtil.cpp [hunk 3]
```diff
@@ -505,14 +504,13 @@ void getStorageConfig(FILE * pOutputFile)
...
 {
-    SystemStatus systemstatus;
     Oam oam;
 
     fprintf(pOutputFile,"System External DBRoot Storage Statuses\n\n");
     fprintf(pOutputFile,"Component     Status                       Last Status Change\n");
     fprintf(pOutputFile,"------------  --------------------------   ------------------------\n");
 
-    try
+    /*try
```
Comment (drrtuy) [inline oamapps/columnstoreSupport/mcsSupportUtil.cpp:513]: Remove the commented section?

File: writeengine/client/we_clients.cpp [hunk 4]
```diff
@@ -417,11 +415,11 @@ void WEClients::Listen ( boost::shared_ptr<MessageQueueClient> client, uint32_t
         }
 
         // send alarm
-        ALARMManager alarmMgr;
+//        ALARMManager alarmMgr;
```
Comment (drrtuy) [inline writeengine/client/we_clients.cpp:418]: Should we remove commented lines?

**Review by drrtuy — APPROVED** [review]

File: writeengine/client/we_clients.cpp (hunk 4 above)
Comment (benthompson15) [reply writeengine/client/we_clients.cpp:418]: Yes, this was just missed by me.

**Pull request merged**